frontend/src/components/UserTypeDropdown.tsx

```diff
@@ -1,22 +1,17 @@
-import { Grid } from '@material-ui/core'
-import { SearchableDropdown, SearchableDropdownOption } from '@equinor/fusion-components'
+import { Select } from '@equinor/fusion-components'
 
-import { HelpIcon } from './HelpIcon'
+type SelectOption = {
+    title: string
+    key: string
+    isSelected?: boolean
+    isDisabled?: boolean
+}
 
 interface Props {
-    userTypeOptions: SearchableDropdownOption[]
+    userTypeOptions: SelectOption[]
     setSelectedUserType: (newSelectedOption: string) => void
 }
 
 export const UserTypeDropdown = ({ userTypeOptions, setSelectedUserType }: Props) => {
-    return (
-        <Grid item container xs={12} spacing={3} alignItems="center">
-            <Grid item xs={10} sm={5} data-testid={'user_type_dropdown'}>
-                <SearchableDropdown label="User type" options={userTypeOptions} onSelect={item => setSelectedUserType(item.title)} />
-            </Grid>
-            <Grid item xs={1}>
-                <HelpIcon helpText={'info text'} />
-            </Grid>
-        </Grid>
-    )
+    return <Select options={userTypeOptions} onSelect={item => setSelectedUserType(item.title)} />
```

Comment:
Maybe this does not have to be defined in a separate file? Maybe it is easier to just place line 16 directly in `Order.tsx` where it is used?